### PR TITLE
Feature/#52 ability to add new members to a flat

### PR DESCRIPTION
**Pull request by style0403**
## Pull Request Information
**Relates to Github Issue:** #52 

**What is the PR Type:** Feature
 

**Describe the added/fixed behaviour:**
Added search and add bar so that the user can add members to flat via username 
image

## Checklist

- [x] Pull request has suitable title, adequately described and associated with a github issue
- [ ] Branch name follows correct convention (e.g. feature/[id]-some-feature)
- [x] Branch builds and executes with no errors
- [ ] Test cases has been generated for the feature and test cases pass
- [ ] Updated relevant wiki/documentation

## Additional Information
currently the only username that works is "test". otherwise it will say it does not have username in the system. 

## Final Reminders
Please remember to squash the commits

### CODE REVIEW

File: Frontend/frontend/.env [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_API="https://localhost:44394/"
\ No newline at end of file
+REACT_APP_BACKEND_API="https://localhost:5001/"
```
Comment (HongShi10) [inline Frontend/frontend/.env]: You should unhardcode it. Maybe set it as an environment variable so other people who run can with different local hosts can change it easily.

Comment (NathanMills1) [reply Frontend/frontend/.env]: This line was not added by him, he just changed the number, probably automatically when running it. It was still hard coded to a different number before hand, and so changing how it works is out of scope of this PR

**Comment by NathanMills1** [comment]
> Can you make sure your branch name follows the right convention. Also, can you reset your local to the remove upstream because im not too sure why your commits contain all previous commits that werent in your branch.

The mishap with previous commits was due to accidentally working from master at the time. It is unfortunate but once its squashed it will only appear as one commit so should be okay

**Comment by style0403** [comment]
> Can you make sure your branch name follows the right convention. Also, can you reset your local to the remove upstream because im not too sure why your commits contain all previous commits that werent in your branch.



> > Can you make sure your branch name follows the right convention. Also, can you reset your local to the remove upstream because im not too sure why your commits contain all previous commits that werent in your branch.
> 
> The mishap with previous commits was due to accidentally working from master at the time. It is unfortunate but once its squashed it will only appear as one commit so should be okay

**Comment by HongShi10** [comment]
you should be able to just checkout to a new branch with the right branch name and create a new PR for that

**Comment by NathanMills1** [comment]
Closed so that it can be properly branched and named before making the PR again.

**Pull request closed without merging**